src/registries/nfts_registry.ts

```diff
@@ -123,7 +123,7 @@ const standardNormaliser = ({
     const userStandardNormalised = standard.toUpperCase();
     const systemStandardNormalised = NFTStandard.dip721.toUpperCase();
     const startsWithDip721 = userStandardNormalised.startsWith(systemStandardNormalised);
-    const hasSuffix = userStandardNormalised.split(systemStandardNormalised).length > 1;
+    const hasSuffix = userStandardNormalised.split(systemStandardNormalised).filter(v => v).length > 0;
```

Comment:
could it happen that `split` returns undefined somehow? If so, I'd add optional chaining `?.filter`

Reply:
@tomiir so it always returns an array, should be alright